### DISCUSSION TITLE
fault don't break surface

### DISCUSSION THREAD

**Discussion by maayange17** [gempy-project/gempy#591] [Q&A] [answered]
I'm trying to create model generator, but in most cases, the fault does not break the surface, why is that?
here is an exemple
![image](https://user-images.githubusercontent.com/47349086/111258372-85b3d800-8625-11eb-9d7c-7d583adc9be9.png)

**Comment by javoha** [comment, marked as answer]
Hi @maayange17,

from what I see in your cross section, your data does not really indicate a significant fault offset. As gempy solves this using an implicit modeling approach and drift functions for faults, the fault offset needs to be inferaable from the data.

I would suggest you try a model with fewer surfaces where there is a clear step from one fault block to the other and check the result.

I will also move this to the discussion section, as I dont think it is a bug.
